Ginger/GingerCoreCommon/Repository/BusinessFlowLib/Activity.cs

```diff
@@ -345,7 +345,7 @@ public string TargetApplication
         /// <summary>
         /// Been used to identify if Acts were lazy loaded already or not
         /// </summary>
-        public bool ActsLazyLoad { get { return mActs.LazyLoad; } }                           
+        public bool ActsLazyLoad { get { return (mActs != null) ? mActs.LazyLoad : false; } }                           
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5022389)